docs/editing-components.md

```diff
@@ -32,9 +32,10 @@ _* If you're going to be doing anything more than making minor updates to
         1. [Adding CSS](#adding-css)
         1. [Adding JavaScript](#adding-javascript)
 1. [How-to guides](#how-to-guides)
-    1. [Adding a field](#adding-a-field)
-    1. [Editing a field](#editing a field)
-    1. [Removing a field](#removing-a-field)
+    1. [Creating a new component](#creating-a-new-component)
+    1. [Adding a field](#adding-a-field-to-an-existing-component)
```

Comment:
Minor suggestion: can these TOC entries contain the full title of the section, e.g. "Adding a field to an existing component?"

Reply:
Sure thing.